### PR TITLE
User story 33 user show

### PR DESCRIPTION
**Pull request by DavidBarriga-Gomez**
**Functionality:**
An admin can now see orders that have status packaged or shipped within their dashboard and can click shipped on a packaged order. Once an order status is shipped a user can no longer cancel that order. 
**Testing:**
Added feature testing for the above functionality. Testing is at 100 percent. 
**Related Issues:**

### CODE REVIEW

**Review by hale4029 — APPROVED** [review]
nice, looks good!

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -39,6 +39,15 @@ def destroy
     Item.increase_item_inventory(order)
   end
 
+  def update
+    # binding.pry
```
Comment (mintona) [inline app/controllers/orders_controller.rb:43]: Comments can be taken out

File: app/views/orders/show.html.erb [hunk 2]
```diff
@@ -51,4 +51,8 @@
   <p>Item Total:<%= @order.total_items %></p>
 </section>
 
-<p><%= link_to "Cancel Order", "/profile/orders/#{@order.id}", method: :delete %></p>
+<% if @order.status == 'shipped' %>
+  <% nil %>
+<% else %>
+  <p><%= link_to "Cancel Order", "/profile/orders/#{@order.id}", method: :delete %></p>
+<% end %>
```
Comment (mintona) [inline app/views/orders/show.html.erb:58]: Could this be said:
<% if @order.status != 'shipped' %>
 <p><%= link_to "Cancel Order", "/profile/orders/#{@order.id}", method: :delete %></p>
<% end %>

**Review by mintona — APPROVED** [review]
Left a comment for a refactor but if it works right now, I'm good with that.

**Pull request merged**